generator/src/main/java/com/scottlogic/deg/generator/fieldspecs/relations/InMapRelation.java

```diff
@@ -27,9 +27,9 @@
 public class InMapRelation implements FieldSpecRelations {
     private final Field main;
     private final Field other;
-    private final DistributedList<String> underlyingList;
+    private final DistributedList<Object> underlyingList;
```

Comment:
por que?

Reply:
The InMap stuff required this, but ideally it should be turned back and the in map handling should be changed instead as it is actually a String list.